### ISSUE TITLE
No filtering through simple text in deck builder?

### ISSUE THREAD

**Issue by lebigot** [seastan/ringsdb#28]
Looking for "Song" in the text filtering field in the deck builder does not filter "Song of…" cards. Instead, no filter is applied. Since the field reads "… or filter the list", I was expecting the Songs to be filtered and shown, only.

**Comment by seastan** [comment]
There is a button just above labeled "filter help" that tells you how the filtering works. Filtering by card title is not one of the available filters because that use case is handled by the dropdown that appears as you type (that's what the "find a card" part of the search field refers to).

**Comment by lebigot** [comment]
I thought that the filter help was only for the non obvious advanced filtering options.

So the dropdown menu is not a filter but a search in the database, so effectively there is no filtering by card name? I guess I can do a simple text search in the browser. 

I was expecting that if I didn't select a name from the dropdown menu, the text would be used as a real card name filter (that filters the list of cards currently displayed in the deck builder), since the field indicates it's a filter. I think that it's an expected behavior, since it's the most basic one (because it's more direct than using the field selectors from the filter help). So that's a feature suggestion. :)

**Comment by seastan** [comment]
Fair enough! Though I think that the dropdown's tendency to cover the filtered results might make this a minor frustration to use.

**Comment by lebigot** [comment]
Good point. Making the menu disappear when the user type enter (or even maybe space) might be a solution?

**Comment by seastan** [comment]
Having it disappear upon space would be super annoying given how many cards include a space. Enter could be a possibility though.

Can you give me some examples of use cases where this would be useful? It might help me come up with a cleaner solution.

**Comment by seastan** [comment]
Yes it does, thanks!
